### PR TITLE
component > Blog Card

### PR DESCRIPTION
**Pull request by evanhameed99**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,9 +1,12 @@
 import React from 'react';
+import BlogCard from './components/Card/BlogCard/BlogCard';
 
 function App() {
   return (
     <div>
-      <h1 className="text-3xl text-purple-500">Train To Maintain</h1>
+      <BlogCard />
```
Comment (osamaakb) [inline src/App.js:7]: you should pass props

File: package.json [hunk 2]
```diff
@@ -7,6 +7,8 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
+    "framer-motion": "^2.2.0",
+    "prop-types": "^15.7.2",
```
Comment (osamaakb) [inline package.json:11]: do we really need this?

File: src/components/Card/BlogCard/BlogCard.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { motion } from 'framer-motion';
+import propTypes from 'prop-types';
+const BlogCard = ({ title, overview, imgUrl, authorName, publishDate }) => {
```
Comment (osamaakb) [inline src/components/Card/BlogCard/BlogCard.js:4]: these should be used not only defined

File: package.json (hunk 2 above)
Comment (evanhameed99) [reply package.json:11]: I am using framer motion library cuz It has a bunch of cool animations and transitions. 
I am using prop-types to validate the props I pass to the card component. If I do not validate the props ,the esLint will always throw warnings! I am really not sure about it! we can definitely talk more about that in our daily stand-up.

**Pull request closed without merging**